### PR TITLE
Added codeowners

### PR DESCRIPTION
**Pull request by SolMendiola**
# :star: Feature

---

#### :pencil2: Description:

Added configuration of codeowners. 

---

### CODE REVIEW

File: .github/CODEOWNERS [hunk 1]
```diff
@@ -0,0 +1 @@
+* @sante @f7deleon @mirland
```
Comment (mirland) [inline .github/CODEOWNERS:1]:
```suggestion
* @sante @f7deleon @mirland

```

Comment (mirland) [reply .github/CODEOWNERS:1]: I'd set the flutter open source team

Comment (f7deleon) [reply .github/CODEOWNERS:1]: didn't we have an issue with this? i find it strange that in donacel and in the template we don't use it

Comment (f7deleon) [reply .github/CODEOWNERS:1]: sure, lets go with both 👍

Comment (f7deleon) [reply .github/CODEOWNERS:1]: you should add `@flutter-open-source` into the file @SolMendiola

**Review by mirland — APPROVED** [review]
Maybe just who is working in the project, the other are added in the group

**Review by f7deleon — APPROVED** [review]

**Pull request merged**